DLaB.CrmSvcUtilExtensions/Entity/CustomizeCodeDomService.cs

```diff
@@ -32,6 +32,10 @@ public void CustomizeCodeDom(CodeCompileUnit codeUnit, IServiceProvider services
             {
                 new CodeCustomization(Parameters).CustomizeCodeDom(codeUnit, services);
             }
+            if(true)
```

Comment:
So the way this works, is you set a flag in the config that its read as a property.  Then I do a search for the app.Config key of a similar value, and find where it all is updated, and create the new property in the UI/Settings of the actual EBG

Reply:
yeah, sounds like a plan, if you want to do that, that would be awesome!
Naming things is hard, so if you have something in mind, then that would be good.
Would it be good if the new config would default to false?

Reply:
I can do it.  When is just the question...

As far as naming things, GenerateConstructorsSansLogicalName?